### PR TITLE
[Origin Trials article] 2nd attempt: add downtime warning

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Adds a warning about downtime.

### CODE REVIEW

File: src/site/content/en/blog/origin-trials/index.md [hunk 1]
```diff
@@ -38,21 +38,31 @@ trials, as learnings are incorporated and adjustments are made.
 ## How to register for an origin trial
 
 1. Choose an origin trial from the [list of active trials](https://developers.chrome.com/origintrials/#/trials/active).
-1. [Request a token](https://developers.chrome.com/origintrials/#/trials/active).
-1. [Add the token](http://googlechrome.github.io/OriginTrials/developer-guide.html)
-   to your web pages, using a `<meta>` tag or an `Origin-Trial` HTTP response
-   header.
+1. Request a token by clicking the **Request** button and filling out the form.
```
Comment (beaufortfrancois) [inline src/site/content/en/blog/origin-trials/index.md:41]: Did you mean "Register" instead of "Request"?

Comment (tomayac) [inline src/site/content/en/blog/origin-trials/index.md:41]:
```suggestion
1. Obtain a token by clicking the **Request** button and filling out the form.
```

Comment (tomayac) [reply src/site/content/en/blog/origin-trials/index.md]: Double "request" is bad style indeed. I went for "obtain" now.

Comment (beaufortfrancois) [reply src/site/content/en/blog/origin-trials/index.md]: I meant the button name, not the verb. Sorry I should have been more clear.

File: src/site/content/en/blog/origin-trials/index.md [hunk 2]
```diff
@@ -38,21 +38,31 @@ trials, as learnings are incorporated and adjustments are made.
 ## How to register for an origin trial
 
 1. Choose an origin trial from the [list of active trials](https://developers.chrome.com/origintrials/#/trials/active).
-1. [Request a token](https://developers.chrome.com/origintrials/#/trials/active).
-1. [Add the token](http://googlechrome.github.io/OriginTrials/developer-guide.html)
-   to your web pages, using a `<meta>` tag or an `Origin-Trial` HTTP response
-   header.
+1. Obtain a token by clicking the **Request** button and filling out the form.
```
Comment (tomayac) [inline src/site/content/en/blog/origin-trials/index.md:41]:
```suggestion
1. Request a token by clicking the **Register** button and filling out the form.
```

**Pull request merged**